release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -2391,11 +2391,125 @@ An update for `jenkins` and `openshift` is now available for {product-title}
+The RPM packages included in the update are provided by the
+link:https://access.redhat.com/errata/RHBA-2020:3511[RHBA-2020:3511] advisory.
+
+Space precluded documenting all of the container images for this release in the
+advisory. See the following article for notes on the container images in this
+release:
+
+link:https://access.redhat.com/solutions/5357141[{product-title} 4.5.8 container image list]
+
+[id="ocp-4-5-8-features"]
+==== Features
+
+[id="ocp-4-5-8-added-projectid-for-network-interfaces"]
+===== Added `projectID` field for network interfaces
+
+The new `projectID` field is now available to configure in a MachineSet custom
+resource under `.spec.template.spec.providerSpec.networkInterfaces`. This field
+allows machines to be booted in shared VPCs.
+
+[source,yaml]
+----
+...
+providerSpec:
+  ...
+  networkInterfaces:
+    - network: <infrastructureID>-network
+      subnetwork: <infrastructureID>-<role>-subnet
+      projectID: <projectID>
+...
+----
+
+For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1868751[*BZ#1868751*].
+
+[id="ocp-4-5-8-bug-fixes"]
+==== Bug fixes
+
+* Previously, intermittent API server errors were reported for the
+`ImageChangesInProgress` condition instead of the `SamplesExists` condition of
+the Samples Operator config object. When the API server reported that all
+samples were installed, the Samples Operator failed to switch the `Progressing`
+condition to `false` because there was unexpected data in its
+`ImageChangesInProgress` condition. This caused upgrades to incorrectly be
+marked as incomplete. This bug fix updates the `SamplesExists` condition to
+report errors on the API server, so upgrades are no longer blocked if
+intermittent API server errors occur while the Samples Operator is upgrading.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1857201[BZ#1857201])
+
+* Previously, the `ironic-image` container configuration was missing the setting
+to enable the `idrac-redfish-virtual-media` boot driver. Because of this, users
+were unable to select the `idrac-virtual-media` boot URL for Metal3. The missing
+`ironic-image` container configuration is now included, so users are able to
+select the `idrac-virtual-media` URL for Metal3.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1859488[BZ#1859488])
+
+* Previously, the Operand form array and object fields did not have logic to
+retrieve and show field descriptions on the form. In result, no descriptions
+were rendered for array or object type fields. This bug fix adds logic to now
+display array and object field descriptions on the Operand creation form.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1861433[BZ#1861433])
+
+* Previously, Buildah erased image architecture and OS fields on images. This
+caused common container tools to fail because the resulting images could not
+identify their architecture and OS. This bug fix forces Buildah to not overwrite
```

Comment:
"forces...to not overwrite" sounds odd to me. If it doesn't change the meaning, I'd suggest something like, "This bug fix prevents Buildah from overwriting...."